packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7979,16 +7981,20 @@ export default class Meeting extends StatelessWebexPlugin {
     }
 
     const permissionTokenExpValue = Number(this.permissionTokenPayload.exp);
+    const permissionTokenIssuedTime = Number(this.permissionTokenPayload.iat);
```

Comment:
maybe it would be worth renaming the variables that store values read from the permission token (iat and exp) so that it's clear that these are server time values not local client time ones